app/views/layouts/application.html.erb

```diff
@@ -117,17 +117,17 @@
     <span class="icons-spinner"></span>
   </div>
 </div>
-
+<script>
+  window._railsEnv = <%= Rails.env.development? || Rails.env.test? %>;
```

Comment:
If `_railsEnv` is defined, it means it's test or development? Isn't more clear to pass the env (`Rails.env`) and behave on code accordingly? 
Or maybe make it something like `pusherConfig = { devMode:  dev? || test? }`

Reply:
I read on somewhere that passing the entire rails.env would smell, since it could have information that shouldn't be available on the frontend. I think the second approach you suggested is more feasible 👍

Reply:
I don't mean passing the entire env, just the env name
```
Rails.env
=> "development"
```